Gemfile

```diff
@@ -17,6 +17,8 @@ else
   gem 'journey'
 end
 
+gem 'i18n', '~> 0.6.11'
```

Comment:
This should only apply on < 1.9.3, so we're otherwise testing with the latest allowed by our gemspec.